### ISSUE TITLE
Don't expand the semi-circle of archived goal when clicked

### ISSUE THREAD

**Issue by Tushar-4781** [tijlleenders/ZinZen#1202]
**Describe the bug**
In archive, the semi-circle is expanding when clicked.

**To Reproduce**
1. Archive a goal 
2. Open Archive drawer 
3. Click on the archived goal 

**Expected behavior**
Since there is no list of actions for a archived goal, we don't want it to expand.

**Screenshots**
![image](https://user-images.githubusercontent.com/39093015/236687701-cd1a846c-3de1-4535-96d4-c8b339bfc2e7.png)

**Desktop (please complete the following information):**
 - OS: Android
 - Browser : chrome

**Comment by tijlleenders** [comment]
Blocked by resolving #1170 first - then this issue might be a non-issue.

**Comment by Tushar-4781** [comment]
@tijlleenders there's no need to block this issue #1170 and this issue is completely isolated. if a dev wants he/she can resolve both of the issues in same PR or in separate PR. its about preventing that circle from expanding

**Comment by tijlleenders** [comment]
**Agree** they can be solved in the same PR.

**Don't agree** they're completely isolated: solving this issue is a waste of time if we're keeping the circle and sub-icons as a part of the solution for #1170

**Comment by tijlleenders** [comment]
This is a super simple fix so if someone wants to solve it without waiting for #1170 go ahead!
